Projects/Coffice/Sources/App/Main/Search/CafeMapView.swift

```diff
@@ -47,6 +47,7 @@ struct CafeMapView: View {
 
           VStack(alignment: .center, spacing: 0) {
             headerView
+              .contentShape(Rectangle())
```

Comment:
reference : https://www.hackingwithswift.com/quick-start/swiftui/how-to-control-the-tappable-area-of-a-view-using-contentshape